x/observer/types/ballot.go

```diff
@@ -93,37 +93,26 @@ func CreateVotes(listSize int) []VoteType {
 
 // BuildRewardsDistribution builds the rewards distribution map for the ballot
 // It returns the total rewards units which account for the observer block rewards
```

Comment:
Outdated, but I would consider still intializing the map in this function and returning it.
Right now we pass an empty map as an argument which is odd

Reply:
The map is initially empty, and values are added since this function is called for each ballot.

Reply:
Yeah but it's a cleaner API then to have the function populating the map to initialize it and return it.